pkg/webhook/validate.go

```diff
@@ -234,7 +235,7 @@ func dynamicValidateSriovNetworkNodePolicy(cr *sriovnetworkv1.SriovNetworkNodePo
 	for _, node := range nodeList.Items {
 		if cr.Selected(&node) {
 			nodesSelected = true
-			err = validatePolicyForNodeStateAndPolicy(nsList, npList, &node, cr)
+			err := validatePolicyForNodeStateAndPolicy(nsList, npList, &node, cr)
```

Comment:
Why is this assignment needed here?

Reply:
Not needed. I'm not sure why that got changed.